tests/test_document_fuse.py

```diff
@@ -28,7 +28,7 @@ def filename_to_ay(filename):
 dir_prefix = "dir"
 ensure_no_remaining = True
 
-running_count = {}
+running_count: dict[str, int] = {}
```

Comment:
slightly unrelated but this is a use-case for a [`defaultdict`](https://docs.python.org/3/library/collections.html#collections.defaultdict)
```py
running_count: dict[str, int] = defaultdict(int)

def get_a_count(counter: str):
    global running_count
    running_count[counter] += 1
    return running_count[counter]
```